tools/sirius_csifingerid/sirius_csifingerid.xml

```diff
@@ -1,15 +1,14 @@
 <tool id="sirius_csifingerid" name="SIRIUS-CSI:FingerID" version="4.0.1+galaxy0.2.6">
```

Comment:
is it possible to use the same version number for the galaxy tool as for the package? i.e just 4.0.1